### ISSUE TITLE
Open Audio & Video Files In EDS Lite

### ISSUE THREAD

**Issue by trymeouteh** [sovworks/edslite#28]
You can open and view images in EDS, please consider support audio and video files as well.

**Comment by pokegamer5547** [comment]
Yes, maybe with exoplayer integration it can be possible to play audio or video without thrid party app. I'm not totally sure though.
